### ISSUE TITLE
Placing a call fails to contact with formatting text in number

### ISSUE THREAD

**Issue by eshattow** [BelledonneCommunications/linphone-android#1744]
Hi!

When I place a call from my contacts list within Linphone it is trying to dial the parenthesis and hyphens. This fails with "User hasn't been found" error.

Dialing their number manually on the keypad all is good and the call is successful.

4. **Please complete the following information** (mandatory)

 - Device: Moto G6
 - OS: Android 9
 - Version of the App Linphone 4.7.0-alpha 365+a08e447d3
 - Version of the SDK 5.2.0-alpha.265+bcfa63d
 - Where you did got it from: snapshots download
 - Please tell us if your Android is a Lineage OS or another variant. Stock Moto Android release.

Will provide logs if needed. Contacts are all sync'ed with DAVx5 from a Nextcloud instance.

**Comment by Viish** [comment]
Hi,

Yes please provide logs showing the issue.

**Comment by eshattow** [comment]
I've tried again with Linphone Android 4.7.0-alpha.388+bb288f03c (pipelines/51152, debug) + Linphone SDK 5.2.0-alpha.280+7a1c2cd (pipelines/51188, release);   The problem is confirmed again on this fresh install of Linphone Android, then change some settings to what I normally have

Audio> true = Adaptive rate control
Audio> openSLES Microphone
Audio> true = Route audio to the bluetooth device if any
Audio> Codec bitrate limit > 15 kbits/s
Audio> Codec>  only G722
Video> false = Enable video
Call> Media encryption > SRTP
Call> true = Media encryption mandatory
Call> true = Improve interactions with bluetooth devices
Call> true = Overlay call notification
Call> Voice mail URI > *97
Chat> false = Hide empty chat rooms
Chat> false = Hide chat rooms from removed proxy configs
Network> false = Use WiFi only
Advanced> Dark theme > Yes
Advanced> Encrypt everything

and search a contact by name in my addressbook from Linphone Android, then attempt to call. That contact has a phone number in the format "(NNN) NNN-NNNN". The error says "User hasn't been found" and briefly I can see the URL being attempted is "sip:(NNN)%20NNN-NNNN@domain.tld"

I used the send logs function and refer to this issue. Thanks.
